app/views/admin/organizations/index.html.haml

```diff
@@ -1,76 +1,76 @@
-    = submit_tag 'Search', name: 'search', class: 'button-blue'
+    .row
+      .col-md-6
+      .col-md-2.align-self-end
+        .form-check.text-center
+          = check_box_tag :is_paid, class: "form-check-input"
+          %label.form-check-label{for: "is_paid"}
+            Paid only?
+      .col-md-3
+        = text_field_tag :query, params[:query], placeholder: "Find name, location #{current_user.developer? ? 'or paid features' : '' }", class: "form-control mt-4"
+      .col-md-1
+        = submit_tag 'Search', name: 'search', class: 'btn btn-primary mt-4'
 
-- if @organizations.present?
-  .paginate-container
-    = paginate @organizations
 
-%section.full-screen-table
-  %table.table.table-striped.table-bordered
-    %thead
-      - ['name', 'approved', 'created_at'].each do |col|
-        - active_col = @sort == col
-        %th
-          %a.column_sort_update{ href: '#', class: ('active' if active_col), style: 'display: block;', data: { sort: col, direction: (active_col && @sort_direction == 'desc' ? 'asc' : 'desc') } }
-            = col.titleize
-            - if col == "created_at"
-              <small class="convertTimezone"></small>
-            - if active_col
-              .pull-right
-                %small
-                  = (@sort_direction == 'asc' ? '&#x2193;' : '&#x2191;').html_safe
+= paginate @organizations, views_prefix: "admin"
 
+.full-screen-table-wrapper
+  %table.table.table-striped.table-bordered
+    %thead.thead-light
+      %th
+        = sortable "name"
+      %th
+        = sortable "approved"
+      %th
+        = sortable "created_at"
       %th
         Bikes
-      %th.medium-screens
+      %th
         Memberships
       %th
         Type
-      %th.large-screens
+      %th
         Locations
     %tbody
       - @organizations.each do |organization|
         - next if organization.deleted_at.present?
         %tr
           %td
             = link_to organization.name, admin_organization_url(organization)
-            %small.less-strong.large-screens
+            %small.less-strong
               = "paid" if organization.is_paid
           %td
             = "&#x2713;".html_safe if organization.approved
-          %td.medium-screens
+          %td
             %span.convertTime
               = l organization.created_at, format: :convert_time
           %td
             = organization.bikes.count
-          %td.medium-screens
+          %td
             = organization.memberships.count
           %td
             = link_to organization.kind.humanize, admin_organizations_path(kind: organization.kind)
-          %td.large-screens
+          %td
             .less-strong-hold
-              
+
               = organization.locations.count
               - if organization.show_on_map
                 %span.less-strong-right
                   Mapped
 
+  = paginate @organizations, views_prefix: "admin"
+
 
 :coffeescript
   $('.column_sort_update').click (e) ->
```

Comment:
This coffeescript was for the old sorting functionality, it shouldn't be in there anymore.